server/validators/stepTargetDateRangeValidator.ts

```diff
@@ -1,14 +1,8 @@
-const MAX_TARGET_DATE_RANGE_LENGTH = 50
```

Comment:
I don't see much point in checking the max length is exceeded, given the values come from radio buttons, plus we'd like to switch to using a enum soon.